addons/base/views.py

```diff
@@ -457,7 +476,9 @@ def create_waterbutler_log(payload, **kwargs):
             node_addon.create_waterbutler_log(auth, action, metadata)
 
     with transaction.atomic():
-        file_signals.file_updated.send(target=node, user=user, event_type=action, payload=payload)
+        if not isinstance(node, Preprint):
```

Comment:
Minor: The if block should be around the transaction, no reason to open it if nothings happening in it.